client/asset/eth/eth_test.go

```diff
@@ -2479,60 +2493,63 @@ func TestMaxOrder(t *testing.T) {
 	redeemerAsset := tBTC
 
 	for _, test := range tests {
-		var assetID uint32 = BipID
-		dexAsset := tETH
-		if test.token {
-			assetID = simnetTokenID
-		}
-
-		w, _, node, shutdown := tassetWallet(assetID)
-		defer shutdown()
+		t.Run(test.name, func(t *testing.T) {
```

Comment:
In `TestPreSwap` you did this thing `runTest := func(t *testing.T, test testData) {`. Can you do the same here?

Reply:
The two tests mirror eachother closely now with that `runTest` pattern